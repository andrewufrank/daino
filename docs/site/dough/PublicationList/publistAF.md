---
title: publistAF.md
author: auf
date: 2010-07-29
keywords: homepage
pageTemplate: page3.yaml
abstract: |
    produce a publist for af

bibliographyGroup: authorAF
style: "/home/frank/Workspace8/ssg/docs/site/dough/resources/chicago-fullnote-bibliography-bb.csl"
bibliography: "/home/frank/Workspace8/ssg/docs/site/dough/resources/BibTexLatex.bib"


---

An example publication list for AF  -- but not the relative "resources/BibTexLatex.bib"

 
