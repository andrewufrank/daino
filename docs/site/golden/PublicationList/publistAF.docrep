Docrep {meta1 = MetaPage {dyFn = "/home/frank/Workspace11/ssg/docs/site/dough/PublicationList/publistAF.md", dyLink = "PublicationList/publistAF.md", dyLang = DLenglish, dyTitle = "publistAF.md", dyAbstract = "produce a publist for af", dyAuthor = "auf", dyDate = Just "2010-07-29", dyKeywords = "homepage", dyBibliography = Nothing, dyStyle = Just "resources/chicago-fullnote-bibliography-bb.csl", dyReferences = Nothing, dyNoCite = Nothing, dyPublish = Nothing, dyIndexPage = False, dyIndexSort = Nothing, dyIndexEntry = IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/PublicationList/publistAF.md", link = "PublicationList/publistAF.md", title = "publistAF.md", abstract = "produce a publist for af", author = "auf", date = "2010-07-29", publish = Nothing, indexPage = False, dirEntries = [], fileEntries = []}}, pan1 = Pandoc (Meta {unMeta = fromList [("abstract",MetaBlocks [Para [Str "produce",Space,Str "a",Space,Str "publist",Space,Str "for",Space,Str "af"]]),("author",MetaInlines [Str "auf"]),("bibliography",MetaInlines [Str "resources/BibTexLatex.bib"]),("bibliographyGroup",MetaInlines [Str "authorAF"]),("date",MetaInlines [Str "2010-07-29"]),("keywords",MetaInlines [Str "homepage"]),("pageTemplate",MetaInlines [Str "page3.yaml"]),("style",MetaInlines [Str "resources/chicago-fullnote-bibliography-bb.csl"]),("title",MetaInlines [Str "publistAF.md"])]}) [Para [Str "An",Space,Str "example",Space,Str "publication",Space,Str "list",Space,Str "for",Space,Str "AF",Space,Str "--",Space,Str "but",Space,Str "not",Space,Str "the",Space,Str "relative",Space,Str "\"resources/BibTexLatex.bib\""]]}