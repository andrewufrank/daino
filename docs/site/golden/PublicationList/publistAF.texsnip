TexSnip {snipyam = MetaPage {dyFn = "/home/frank/Workspace11/ssg/docs/site/dough/PublicationList/publistAF.md", dyLink = "PublicationList/publistAF.md", dyLang = DLenglish, dyTitle = "publistAF.md", dyAbstract = "produce a publist for af", dyAuthor = "auf", dyDate = Just "2010-07-29", dyKeywords = "homepage", dyBibliography = Nothing, dyStyle = Just "resources/chicago-fullnote-bibliography-bb.csl", dyReferences = Nothing, dyNoCite = Nothing, dyPublish = Nothing, dyIndexPage = False, dyIndexSort = Nothing, dyIndexEntry = IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/PublicationList/publistAF.md", link = "PublicationList/publistAF.md", title = "publistAF.md", abstract = "produce a publist for af", author = "auf", date = "2010-07-29", publish = Nothing, indexPage = False, dirEntries = [], fileEntries = []}}, unTexSnip = "An example publication list for AF -\\/- but not the relative\n\"resources/BibTexLatex.bib\""}