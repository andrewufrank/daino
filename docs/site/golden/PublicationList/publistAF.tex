\documentclass[a4paper,10pt]{scrbook}
\usepackage{fontspec}
\usepackage[ngerman]{babel}
\usepackage{graphicx}
\usepackage{makeidx}
\usepackage{natbib}
\makeindex
\usepackage[colorlinks]{hyperref}
\providecommand{\tightlist}{%
\setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}

\begin{document}

An example publication list for AF -\/- but not the relative
"resources/BibTexLatex.bib"


\printindex
\end{document}
